theme_monglia/views/customizations.xml

```diff
@@ -847,4 +908,122 @@
+        <attribute name="data-file-name">uiface_2.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
+    </xpath>
+    <!-- Member 3 -->
+    <xpath expr="(//h3)[3]" position="replace">
+        <h3 style="text-align: center;"><span class="o_text_highlight o_translate_inline o_text_highlight_bold_1 o_text_highlight_fill" style="--text-highlight-width: 3px; --text-highlight-color: #AFF8C8;"><span class="o_text_highlight_item">Aline Turner<svg fill="none" class="o_text_highlight_svg o_content_no_merge position-absolute overflow-visible top-0 start-0 w-100 h-100 pe-none"><path stroke-width="var(--text-highlight-width)" stroke="var(--text-highlight-color)" stroke-linecap="round" fill="var(--text-highlight-color)" transform="scale(1.2715234375, 1)" d="M190.276 34.01c5.618-.25 7.136-6.526 4.444-9.755.037-.25.055-.5.072-.749 7.046-.949 7.01-11.752-.523-11.553-.796.017-1.59.017-2.403.05C196.78 9.573 195.931.8 189.264.983L13.784 5.678c-7.226.2-7.497 9.422-1.499 11.32-2.186 0-4.354 0-6.54-.017-7.696-.05-7.624 11.286 0 11.635 8.22.383 16.423.733 24.643 1.016l-7.823.35c-7.624.349-7.678 11.985 0 11.635 55.915-2.53 111.813-5.077 167.729-7.607h-.018Z" class="o_text_highlight_path_bold_1"></path></svg></span></span></h3>
+    </xpath>
+    <xpath expr="(//img)[3]" position="attributes">
+        <attribute name="src">/web_editor/image_shape/website.s_company_team_image_3/web_editor/geometric_round/geo_round_diamond.svg</attribute>
+        <attribute name="data-shape">web_editor/geometric_round/geo_round_diamond</attribute>
+        <attribute name="data-original-mimetype">image/jpeg</attribute>
+        <attribute name="data-file-name">uiface_3.svg</attribute>
+        <attribute name="data-shape-colors">;;;;</attribute>
+    </xpath>
+    <!-- Member 4 -->
+    <xpath expr="(//h3)[4]" position="replace">
+        <h3 style="text-align: center;"><span class="o_text_highlight o_translate_inline o_text_highlight_bold o_text_highlight_fill" style="--text-highlight-width: 3px;"><span class="o_text_highlight_item">Iris Joe<svg fill="none" class="o_text_highlight_svg o_content_no_merge position-absolute overflow-visible top-0 start-0 w-100 h-100 pe-none"><path stroke-width="var(--text-highlight-width)" stroke="var(--text-highlight-color)" stroke-linecap="round" fill="var(--text-highlight-color)" transform="scale(0.7835546875, 0.9130434782608695)" d="M136.604 41.568c5.373.513 10.746 1.047 16.12 1.479 14.437 1.13 29.327 4.047 42.858-4.294 4.92-3.04 2.346-13.56-2.687-13.395-.825.02-1.635.062-2.46.082.858-3.677-.34-8.3-3.545-9.41 2.655.062 5.309.104 7.963.165 6.863.185 6.863-14.176 0-14.36A1958.994 1958.994 0 0 0 5.263 5.778C-.4 6.169-2.392 18.455 3.84 19.893c9.727 2.24 19.454 4.335 29.214 6.307-1.085 1.09-1.764 2.671-2.023 4.356-.615.061-1.214.102-1.83.164-6.748.74-6.959 14.587 0 14.361l107.42-3.513h-.016Z" class="o_text_highlight_path_bold"></path></svg></span></span></h3>
```

Comment:
Only the wrapping span for this please, no inline span, no inline svg

Reply:
@xO-Tx I am merging this PR with https://github.com/odoo/design-themes/pull/969 -> I removed the inner svg etc, but I have issues. Could you check the monglia "team shape" snippet once it is done? For some reason:
- The preview in the snippet dialog is good
- Once dropped in the page, only the first highlight works
- Once I save the page, all the highlights work